app/uk/gov/hmrc/transitmovementsrouter/models/ObjectStoreFileDirectory.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.transitmovementsrouter.models
+
+import uk.gov.hmrc.objectstore.client.Path
+
+case class ObjectStoreFileDirectory(value: Path.File) extends AnyVal
```

Comment:
Similarly, `Path.File` is already a specific type for object-store. No need to do this.

Remove this `AnyVal`